### PR TITLE
Added initial Android skeleton app

### PR DESCRIPTION
**Pull request by riggaroo**
### Description of Changes:

We need a base Android project to work on, this PR introduces a skeleton for the Android project, no working functionality just yet. Just the base application that IntelliJ generates for ViewModel+Fragment. 

### Tested on:
Ran build from intelliJ and deployed to device.

### CODE REVIEW

File: android/.idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+TF Segement + Stylize
```
Comment (margaretmz) [inline android/.idea/.name]: Typo - "Segement" should be "Segment".  Later we can perhaps come up with a better app name.

File: android/app/src/androidTest/java/org/tensorflow/example/tfsegementstylize/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```
Comment (RivuChk) [inline android/app/src/androidTest/java/org/tensorflow/example/tfsegementstylize/ExampleInstrumentedTest.kt]: do we want to keep this file?

File: android/app/src/test/java/org/tensorflow/example/tfsegementstylize/ExampleUnitTest.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```
Comment (RivuChk) [inline android/app/src/test/java/org/tensorflow/example/tfsegementstylize/ExampleUnitTest.kt]: Do we want to keep this file?

File: android/settings.gradle [hunk 4]
```diff
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "TF Segement + Stylize"
```
Comment (RivuChk) [inline android/settings.gradle]: As @margaretmz suggested, "Segement" -> "Segment", we can comeup with a proper name later.

**Review by RivuChk — APPROVED** [review]
All my review comments are nonsubstantial, and we can come back to them later, so approved the PR

File: android/.idea/.name (hunk 1 above)
Comment (riggaroo) [reply android/.idea/.name]: 🙈 Missed this, thanks - have fixed it and pushed.

File: android/app/src/androidTest/java/org/tensorflow/example/tfsegementstylize/ExampleInstrumentedTest.kt (hunk 2 above)
Comment (riggaroo) [reply android/app/src/androidTest/java/org/tensorflow/example/tfsegementstylize/ExampleInstrumentedTest.kt]: Deleted 💯

File: android/app/src/test/java/org/tensorflow/example/tfsegementstylize/ExampleUnitTest.kt (hunk 3 above)
Comment (riggaroo) [reply android/app/src/test/java/org/tensorflow/example/tfsegementstylize/ExampleUnitTest.kt]: Deleted 💯

File: android/settings.gradle (hunk 4 above)
Comment (riggaroo) [reply android/settings.gradle]: Fixed :100:

**Review by margaretmz — APPROVED** [review]

**Pull request merged**